demo/campaign/src/blocks/ImageBlock.tsx

```diff
@@ -0,0 +1,16 @@
+import { PropsWithData } from "@comet/cms-site";
+import { MjmlColumn } from "@luma-team/mjml-react";
+import { PixelImageBlockData } from "@src/blocks.generated";
+import { CommonImageBlock } from "@src/common/blocks/CommonImageBlock";
+import { IndentedSectionGroup } from "@src/components/IndentedSectionGroup";
+import { theme } from "@src/util/theme";
+
+export const ImageBlock = ({ data }: PropsWithData<PixelImageBlockData>) => {
+    return (
+        <IndentedSectionGroup paddingLeft={0} paddingRight={0}>
```

Comment:
If you don't want the indentation, please use `MjmlSection` instead. 
It does have a default-padding but that should probably be removed globally, like here: ...

Reply:
> It does have a default-padding but that should probably be removed globally, like here: https://github.com/vivid-planet/comet-brevo-module/pull/57#discussion_r1721267201

This is still missing, right? 
There is now a spacing above and below the image:

<img width="765" alt="Screenshot 2024-08-19 at 10 17 49" src="https://github.com/user-attachments/assets/7028712f-6727-4006-8155-4532765b9c1d">

Reply:
I think, I got it now - thank you @jamesricky  👍